package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "5.6.42",
+  "version": "5.7.0",
```

Comment:
This is the start of a new feature, but not an Earth-shattering change to this UI, so I think increasing the minor release is acceptable.

Ref: https://semver.org/